config/rbac/role.yaml

```diff
@@ -143,6 +143,13 @@ rules:
   - list
   - patch
   - watch
+- apiGroups:
+  - kmm.sigs.x-k8s.io
+  resources:
+  - nodemodulesconfigs/finalizers
```

Comment:
1. https://github.com/rh-ecosystem-edge/kernel-module-management/pull/741/files#diff-4abf41a7395eeba9459df6b2fe59c88e93e33e83dd683d40878c61651de912f6R50
2. This is needed because OpenShift enables a feature gate that is optional upstream, e.g. it's off in minikube. I would not update upstream.